doc/examples/reconst_dti.py

```diff
@@ -300,6 +300,6 @@
 ----------
 .. footbibliography::
 
-.. include:: ../links_names.inc
+.. include:: ../../links_names.inc
```

Comment:
to remove, this is automatically added by the generation script.  a forgotten case

Reply:
Removed in aa6db82.